app/models.py

```diff
@@ -0,0 +1,153 @@
+from app import db
+from datetime import datetime
+
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(50), nullable=False)
+    surname = db.Column(db.String(50), nullable=False)
+    email = db.Column(db.String(100), unique=True, nullable=False)
+    password = db.Column(db.String(100), nullable=False)
```

Comment:
Do we store the passwords? In plain text? :see_no_evil: :scream:

Reply:
of course not, that will be hash, even though the name doesn't suggest it
so I will change the name to make it clear

Reply:
Also, email seems to me equally important as telephone number (in terms of privacy). So we should either remove both from the `__repr__` or we include them both. 

What is your opinion? (personally, I would stick with having them both in `__repr__`)